morango/models.py

```diff
@@ -10,78 +12,72 @@
 ###################################################################################################
 class SyncableModelQuerySet(models.query.QuerySet):
 
-    def update(self, **kwargs):
-        kwargs.update({'_dirty_bit': True})
+    def update(self, dirty_bit_signal=True, **kwargs):
```

Comment:
Since "signal" has a specific meaning in Django, might be better to call it something like `update_dirty_bit_to`?